src/classLibrary/interfaces/interfaces.ts

```diff
@@ -2,11 +2,14 @@ import { of, Union } from "ts-union";
 
 type Syntax = [string,string] // [open, close] , [ ["~", "~"] , ["{{"], ["}}"]]
 
+type CyanFlag = { [s:string] : boolean | CyanFlag }
+type  CyanVariable = { [s:string] : string | CyanVariable }
```

Comment:
Just small nitpick! Extra spacing here

Reply:
We can ignore formatting, I'll setup auto-formatting stuff soon